docs/custom-plugins/custom-rules.md

```diff
@@ -40,13 +40,15 @@ Adding this as part of a plugin requires you to add it to the `rules` part of th
 ```js
```

Comment:
I think the previous paragraph needs to be updated to reflect the code changes, can we check it still makes sense?

Reply:
Updated